executor/runtime/types/container.go

```diff
@@ -378,7 +385,7 @@ func addProcessLabels(containerInfo *titus.ContainerInfo, labels map[string]stri
 
 		command := process.GetCommand()
 		if command != nil {
-			commandStr := strings.Join(entryPoint[:], " ")
+			commandStr := strings.Join(command[:], " ")
```

Comment:
@gabrielhartmann - turns out this has been a bug in this code all along.  Will fixing what gets put into the command label mess up something in titus-isolate or the training models?

Reply:
Huh. We have this internal ticket TITUS-4150 (deleted?) that sounds super similar.

Reply:
CC: @rostyboost, in case you can weigh in here.

Reply:
LGTM.  I don't think it will screw any training severely, but we should confirm with @rostyboost

Reply:
This should be ok.